c2cgeoportal/tests/functional/test_entry.py

```diff
@@ -77,12 +93,27 @@ def setUp(self):  # noqa
         main = Interface(name=u"desktop")
         mobile = Interface(name=u"mobile")
 
+        engine = sqlahelper.get_engine()
+        init(engine)
+        engine.connect()
+
+        a_geo_table = Table(
+            "a_geo_table", declarative_base(bind=engine).metadata,
+            Column("id", types.Integer, primary_key=True),
+            Column("geom", Geometry("POINT", srid=21781)),
+            schema="main"
+        )
+
+        self._tables = [a_geo_table]
+        a_geo_table.drop(checkfirst=True)
+        a_geo_table.create()
+
         public_layer = LayerV1(name=u"__test_public_layer", public=True)
         public_layer.is_checked = False
         public_layer.interfaces = [main, mobile]
 
         private_layer = LayerV1(name=u"__test_private_layer", public=False)
-        private_layer.geo_table = "a_schema.a_geo_table"
+        private_layer.geo_table = "main.a_geo_table"
```

Comment:
I think that all editable table should not be n the main schema as it will be in real usage.